pkg/reconciler/service/service_test.go

```diff
@@ -46,12 +49,41 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgotesting "k8s.io/client-go/testing"
+	serviceconfig "knative.dev/serving/pkg/reconciler/service/config"
 
 	. "knative.dev/pkg/reconciler/testing"
 	. "knative.dev/serving/pkg/reconciler/testing/v1"
 	. "knative.dev/serving/pkg/testing/v1"
 )
 
+const (
+	TestIngressClass      = "ingress-class-foo"
+	defaultDomainTemplate = "{{.Name}}.{{.Namespace}}.{{.Domain}}"
+)
+
+type testConfigStore struct {
+	config *serviceconfig.Config
+}
+
+func (t *testConfigStore) ToContext(ctx context.Context) context.Context {
+	return serviceconfig.ToContext(ctx, t.config)
+}
+
+var _ pkgreconciler.ConfigStore = (*testConfigStore)(nil)
+
+func ReconcilerTestConfig(enableAutoTLS bool) *serviceconfig.Config {
```

Comment:
Why does this need to take enbleAutoTLS? All the service should care about from the network config is DefaultIngress.

Reply:
Removed.